### ISSUE TITLE
Diversity Claim

### ISSUE THREAD

**Issue by GmanB3398** [matloff/TidyverseSkeptic#13]
On your paragraph about diversity, I am not sure where this claim that people say that tidy is mainly for making it easier for women and minorities. This (strawman) argument seems incongruous with your evidence based approach you use in the rest of the essay. I would reconsider leaving this paragraph in, as it goes against the exact point you make in the next section.

**Comment by matloff** [comment]
Actually, the claim that Tidy has brought large numbers of women and minorities to R
has come up many times in the last couple of weeks on Twitter. It has also been stated before by people affiliated with R-Ladies and, for that matter, connected to RStudio, both publicly and to me in private.

**Comment by DianeBeldame** [comment]
Unfortunately, anecdote is not data

**Comment by matloff** [comment]
An essay is not a research paper.

**Comment by DianeBeldame** [comment]
That's not an excuse for argumentum ad verecundiam

**Comment by pmarchand1** [comment]
That part of the R community was found to be more inclusive by many people, and that the tools were found to be easier to learn, doesn't imply what you're saying in the essay. Maybe the same people are making both pedagogical efforts and efforts at creating a more positive community at the same time.

**Comment by matloff** [comment]
GmanB3398, it wouldn't matter how footnotes, citations and so I would include in my essay. The RStudio loyalists wouldn't accept it. I've expressed my thoughts here, and that's it, no aspiration for proselytizing.

**Comment by matloff** [comment]
My point about tapply() was that the so-called "base-R solution" for "Summarise rows within groups" appears (sorry, only skimmed it) to be a typical tapply() situation, far simpler than the claimed "base-R solution" and certainly simpler and more obvious to code.

**Comment by drag05** [comment]
@GmanB3398 
I agree with your point that the diversity argument seems a bit forced. 
@DianeBeldame
I think anecdote *is* data and I don't appreciate the veiled push for twitter as being an authority.
@matloff
Beautiful article! Teachability is a valid entry point to this more insidious internal split (takeover) of `R`.

**Comment by DianeBeldame** [comment]
@drag05 I don't care about what you think and I don't care if you appreciate anything I've written **or not**. 

There is evidence that the R community is more diverse than others in STEM. Period. 
And : 
- correlation is not causality
- anecdote is not data. Period again. 

Writing an essay doesn't mean armchair philosophy. 

Thing is, I run an R training organisation. We've trained +2000 adults so far in small groups (half a dozen) for +20k hours*learners and I have evidence that Rmd first + tidyverse first + documentation first approaches combined to teaching techniques coming from neurolearning improves learning experience and upskilling. 

Almost forgot...we don't teach for loops or matrix since 2018. Yet, our learners build packages on a daily basis as naturally as breathing to ship them in production. I would love to tell you more about `$` but you're not ready...

It's not because you've learnt it the hard way you should do the same and perpetrate misconceptions and unfounded beliefs. 

But it's common wisdom that @matloff is more prompt to criticize than reassess himself : 'You can lead a horse to water but you can’t make him drink'

**Comment by drag05** [comment]
@DianeBeldame : Just take a deep breath ...

**Comment by matloff** [comment]
Well, @DianeBeldame, I've been reassessing myself all my life. :-) I've gone from FORTRAN to C to Perl to Python to S to R, both in teaching and in my own use. Today I use/teach mainly R and Python. I've also been a huge supporter of RStudio, both the product and the RS staff, from the very beginning. Accusing someone of inability to change is a standard method of attack, but it doesn't work here, sorry. I hope you'll avoid ad hominem.

**Comment by aphalo** [comment]
@DianeBeldame @matloff In my experience the teachers' view of what is difficult or not very easily is taken up by his/hers students. So, this is something to consider in this discussion. Another, more important question is this: knowing the tidyverse is a very different thing to knowing the R language. The tidyverse has become a new language or grammar built on top of R. To me someone who does not know about R's for loop construct or the various apply functions does not know the R language, point. If I interpret Matloff's view correctly he says that starting by teaching the tidyverse is not a good way of teaching the R language. Something, I agree with. @DianeBeldame In spite of your strong words, in my view your own attack confirms this: you are stopping short of teaching your students the R langauge! You teach them the tidyverse.

**Comment by matloff** [comment]
Yes, my primary concern is for new learners of R.

**Comment by drag05** [comment]
@matloff 

> "*Yes, my primary concern is for new learners of R*"
 
.... and as Teacher, you give them the opportunity of making an informed choice while majority of tidyverse teachers - as their stance and language used in this thread illustrate - resent it .  

One could suspect an aliasing pecuniary factor here. Maybe this is one case where *causation* (sic) is correlation since base R requires fewer lessons to master.

**Comment by ratnanil** [comment]
This sums up everything which is wrong if you're dogmatic about the tidyverse (or anything, really):

> Almost forgot...we don't teach for loops or matrix since 2018. [...] I would love to tell you more about $ but you're not ready...

**Comment by drag05** [comment]
@ratnanil Yeah! Will I ever be ready for $$$$ !? For sure @DianeBeldame could share some tips
